docusaurus/docs/iOS/guides/working-with-custom-attachments.md

```diff
@@ -0,0 +1,352 @@
+---
+title: Working with Attachments in Composer
+---
```

Comment:
Should we stop using the "Working with" pattern? I think it is very redundant that for every guide we start with this naming. For example on React Native I think it is much better and simpler:
![image](https://user-images.githubusercontent.com/12814114/123785773-d39cd500-d8d0-11eb-86c0-22794147703c.png)